sdk/cosmos/azure-cosmos/README.md

```diff
@@ -750,6 +750,52 @@ not being able to recognize the new NonStreamingOrderBy capability that makes ve
 If this happens, you can set the `AZURE_COSMOS_DISABLE_NON_STREAMING_ORDER_BY` environment variable to `"True"` to opt out of this
 functionality and continue operating as usual.*
 
+### Public Preview - Full Text Policy and Full Text Indexes
+We have added new capabilities to utilize full text policies and full text indexing for users to leverage full text search
+utilizing our Cosmos SDK. These two container-level configurations have to be turned on at the account-level
+before you can use them.
+
+A full text policy allows the user to define the default language to be used for all full text paths, or to set
+a language for each path individually in case the user would like to use full text search on data containing different
+languages in different fields.
+
+A sample full text policy would look like this:
```

Comment:
what format is the lnaguage expected to be in I assume <L>-<C> (with <L> being the ISO-639 lanugauge code and <C> the ISO-3166 Country code? Are there plans to support some special-casing for neutral or similar? What about use cases where  a customer does not know the exact language - can they just use en (wnglish independent of country)?